### PR TITLE
fix: replace uib-btn-radio by oui-radio

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
The uib-btn-radio component from bootstrap has been replaced by by oui-radio

MANAGER-4055

### CODE REVIEW

File: packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/telephony-scheduler.less [hunk 1]
```diff
@@ -48,6 +48,22 @@
   }
 }
 
+.telecom-scheduler-bank-holidays {
+  .year-list .oui-radio-toggle-group {
+    flex-direction: column;
+
+    .oui-radio-toggle,
+ .oui-radio-toggle:first-of-type,
```
Comment (FredericEspiau) [inline packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/telephony-scheduler.less]: Identation

File: packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/telephony-scheduler.less [hunk 2]
```diff
@@ -48,6 +48,22 @@
...
+ .oui-radio-toggle:last-of-type {
```
Comment (FredericEspiau) [inline packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/telephony-scheduler.less]: Why is this needed?

File: packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/actions/bankHolidays/telephony-scheduler-bank-holidays.html [hunk 3]
```diff
@@ -42,15 +42,13 @@
...
-                        <button class="btn btn-default"
-                                type="button"
-                                data-ng-class="{ active: SchedulerHolidaysCtrl.model.year === year }"
-                                data-ng-repeat="year in SchedulerHolidaysCtrl.yearList track by $index"
-                                data-ng-model="SchedulerHolidaysCtrl.model.year"
-                                data-uib-btn-radio="'{{ year }}'">
-                            <span data-ng-bind="year"></span>
-                            <span data-ng-bind="'(' + SchedulerHolidaysCtrl.filterEvents(year).length + ')'"></span>
-                        </button>
+                        <oui-radio-toggle-group class="year-list" data-model="SchedulerHolidaysCtrl.model.year">
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/actions/bankHolidays/telephony-scheduler-bank-holidays.html]:
```suggestion
                        <oui-radio-toggle-group data-model="SchedulerHolidaysCtrl.model.year">
```
I don't see the class being used originally

File: packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/telephony-scheduler.less (hunk 1 above)
Comment (ganeshkumar1989) [reply packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/telephony-scheduler.less]: This is done

File: packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/telephony-scheduler.less (hunk 2 above)
Comment (ganeshkumar1989) [reply packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/telephony-scheduler.less]: The UI originally had the `uib-btn-radio` displayed as a column (meaning the buttons, which denotes the years like 2018, 2019 etc are displayed one below the other).  But the `oui-radio-toggle` component does not support this vertical orientation. Only horizontal orientation of the buttons is supported, due to which, the borders are not visible for the right side of the first button, both sides of the buttons in the middle and the left side of the last button. Here we changed the orientation of the buttons to vertical and hence we will have to have uniform borders for each of the buttons, which is what has been done :)

File: packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/actions/bankHolidays/telephony-scheduler-bank-holidays.html (hunk 3 above)
Comment (ganeshkumar1989) [reply packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/actions/bankHolidays/telephony-scheduler-bank-holidays.html]: The UI originally had the `uib-btn-radio` displayed as a column (meaning the buttons, which denotes the years like 2018, 2019 etc are displayed one below the other).  But the `oui-radio-toggle` component does not support this vertical orientation. This class `year-list` has been added to change the orientation of the buttons to vertical and to handle some css changes, that have been explained in the comment above.

Comment (marie-j) [reply packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/actions/bankHolidays/telephony-scheduler-bank-holidays.html]: Maybe instead of adding custom styles we could see if it makes sense to add that as an option to `oui-radio-toggle-group`.

Comment (ganeshkumar1989) [reply packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/actions/bankHolidays/telephony-scheduler-bank-holidays.html]: Point taken. A new ticket is created for this: UK-125. Once done, the manager can be updated with the new option
